apps/frontend/src/pages/projects/projects.tsx

```diff
@@ -39,8 +39,20 @@ const Projects = (): JSX.Element => {
 	const [selectedProject, setSelectedProject] =
 		useState<null | ProjectGetAllItemResponseDto>(null);
 
-	const { dataStatus, projectCreateStatus, projectPatchStatus, projects } =
-		useAppSelector(({ projects }) => projects);
+	const {
+		dataStatus,
+		project,
+		projectCreateStatus,
+		projectPatchStatus,
+		projects,
+		projectStatus,
+	} = useAppSelector(({ projects }) => projects);
+
+	useEffect(() => {
+		if (selectedProject !== null) {
```

Comment:
Maybe we need to save only project id in this case? and let's rename it to projectToModifyId (it's ok if we will use it for deletion too). That's how we named these variables in similar cases
``` suggestion
		if (projectToModifyId !== null) {
```